<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrueTables</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="TabPane">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>1071</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabSetData">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLineEdit" name="atomos_input">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Atomo">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Atomos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addAtom">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="list_atomos">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Premisas">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>110</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Premisas</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="premisas_input">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>100</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addPremisa">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>100</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="list_atomos_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>140</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabTable">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSet_Table"/>
    <addaction name="actionGenerate_Table"/>
   </widget>
   <widget class="QMenu" name="Exit">
    <property name="title">
     <string>Exit</string>
    </property>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="File"/>
   <addaction name="Exit"/>
  </widget>
  <action name="actionSet_Table">
   <property name="text">
    <string>Set Table</string>
   </property>
  </action>
  <action name="actionGenerate_Table">
   <property name="text">
    <string>Generate Table</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
